LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/CupsTwinInfo.cs

```diff
@@ -7,33 +7,49 @@ namespace LoRaWan.NetworkServer.BasicsStation
 {
     using System;
     using System.Text.Json.Serialization;
+    using Newtonsoft.Json;
```

Comment:
It looks like we use both system.text and newtonsoft? Is there a reason for that?

Reply:
Yes, unfortunately TwinCollection object is only working with Newtonsoft.Json JObjects, therefore we need this at the moment for the E2E tests to be able to dynamically change the twins on the fly